### ISSUE TITLE
[Documentation] Very difficult to find list of all mpv.conf options, via Google web search

### ISSUE THREAD

**Issue by jimbo1qaz** [mpv-player/mpv#6561]
It was impossible for me to find a list of all mpv.conf options.

The first 4 Google results (my Google search history is disabled) when searching `mpv.conf documentation` are:

- https://mpv.io/manual/stable/
    - wrong title, it says "pseudo gui mode" but is a complete manual with CLI flags
    - but CLI flags are buried quite a way down
    - I tried ctrl+f mpv.conf, and got to https://mpv.io/manual/stable/#location-and-syntax (which has a similar blurb as this file, but doesn't explain how to find all options at all, and lacks the paragraph at line 15, which I block-quoted below).
    - Had I thought to ctrl+f `sub-font`, I would've found a reference to `--sub-font-size` and the heading of `--sub-font`.
    - But I didn't think to do that, instead I searched `sub-font` on Github source code search, and somehow got zero relevant hits :(
    - Searching mpv's Github, I found https://github.com/mpv-player/mpv/blob/master/DOCS/man/mpv.rst#location-and-syntax which is like https://mpv.io/manual/stable/#location-and-syntax , but missing a list of CLI flags.
- https://github.com/mpv-player/mpv/blob/master/etc/mpv.conf (this file, provides no link to all MPV options)
    - Actually, line 15 in this file contains a long paragraph:

        >Configuration file settings and the command line options use the same underlying mechanisms. Most options can be put into the configuration file by dropping the preceding '--'. See the man page for a complete list of options.

        But my PR will put this information concisely, at the front and center of this file. (Also `mpv --help` or `--list-options`?)

- https://github.com/Argon-/mpv-config/blob/master/mpv.conf (some random user's personal config file)
- https://wiki.archlinux.org/index.php/mpv (provides no link to all MPV options). The first search result to even mention a list of all options is http://manpages.ubuntu.com/manpages/bionic/man1/mpv.1.html (a link to mpv's entire manpage).

The first 3 Google results for `mpv.conf all options` all appear in the above list. https://github.com/mpv-player/mpv/blob/master/etc/mpv.conf is the first result, which I am editing in this PR.

--------

Only after posting in a Discord server was I told that mpv.conf options are identical to CLI options, and I could find all mpv.conf keys by running `mpv --help` or looking at the manpage, then noticing the `--list-options` flag.

I will be submitting 1 or 2 documentation pull requests soon.

**Comment by lachs0r** [comment]
Wow fuck, it’s like you gotta read the manual in order to understand the manual!


Jesus, what is happening to people’s attention span.

**Comment by jimbo1qaz** [comment]
wow, rude @lachs0r 

Did you even read my entire post? Look at the issues I pointed out? Notice that the manual on the mpv website has the wrong title? Look at my pull request https://github.com/mpv-player/mpv/pull/6562 ?

**Comment by lachs0r** [comment]
The title is a Google bug, not our problem. Plus Google results are gonna be different for each person, depending on tracking, location, browser, language, etc. The rest is self-explanatory if you just have the patience to read a few paragraphs, or even just the damn headings. There’s one right at the start of the CONFIGURATION FILES section that says “Putting Command Line Options into the Configuration File”.

**Comment by Argon-** [comment]
If I get it correctly, the whole confusion revolves around the fact that all CLI options are also mpv.conf options and this relationship is not obvious to a new user?
The manual explicitly mentions this:
>You can put all of the options in configuration files which will be read every time mpv is run.
You explicitly linked to this part of the manual yourself, so I don't really get why there was still confusion afterwards.
Not sure about your problem with the pseudo GUI chapter.

**Comment by jimbo1qaz** [comment]
>The title is a Google bug, not our problem.

https://mpv.io/manual/stable/ contains `<title>mpv.io</title>`. Maybe you should add some meaningful information to the title, like "Manual - mpv.io", so Google doesn't have to incorrectly guess at meaningful page titles? Why is mpv one of those unhelpful websites which use the same title for every page in the site?

>Google results are gonna be different for each person, depending on tracking, location, browser, language, etc.

One person on Discord got the same result and same wrong title at a slightly different search position.

>There’s one right at the start of the CONFIGURATION FILES section

which is a full page down on my 1080p 1.3x DPI laptop display.

Even then, it says:

>Almost all command line options can be put into the configuration file. Here is a small guide:

and does not point to any command to list all command line options.

**Comment by CounterPillow** [comment]
>Again, the rudeness is absolutely unprofessional and uncalled for.

Welcome to FOSS multimedia development, buddy. Nobody here is under any obligation to fake niceness, hence you'll get yelled at a lot.

**Comment by lachs0r** [comment]
The manual itself is a list of all options, along with explanations of what they do (which, let’s be honest, you’re definitely gonna need). I don’t see what the problem is.

**Comment by jimbo1qaz** [comment]
>Nobody here is under any obligation to fake niceness, hence you'll get yelled at a lot.
>self-explanatory

and i have no obligation to think the developers are not arrogant and hostile. Actually I've pointed out a mistake I made, which renders my pull request partly unnecessary (mpv.conf mentions --list-options, the mpv.conf manual does not)... but the developers didn't notice. Along the way I've noticed other issues (like missing per-page title on the website) which the developers have yet to respond constructively to.

**Comment by lachs0r** [comment]
It’s not that we didn’t notice, it’s that we didn’t even waste any time reviewing your change yet, because basically what you’re asking is that we restructure and reorder documentation so that the sentences you happen to be looking for right now appear in the order you expect so you don’t have to read more than a dozen words at a time. This is going nowhere. You need to stop working so hard to avoid spending time to read and understand, or you will end up wasting everybody else’s.

**Comment by jimbo1qaz** [comment]
>restructure and reorder documentation

i added 2 sentences.

**Comment by CounterPillow** [comment]
THE MANUAL LISTS THE OPTIONS TOO WHAT THE HELL IS YOUR ISSUE THERE IS A SIDEBAR OF HEADINGS ON THE LEFT WITH OPTIONS AS A HEADING HOLY MACARONI AAAAAAAAAAAAAAA YOU JUST TOOK TEN YEARS OFF MY LIFE EXPECTANCY WITH YOUR ENTITLED BULLSHIT

**Comment by jimbo1qaz** [comment]
>THE MANUAL LISTS THE OPTIONS TOO WHAT THE HELL IS YOUR ISSUE THERE IS A SIDEBAR OF HEADINGS ON THE LEFT WITH OPTIONS AS A HEADING HOLY MACARONI AAAAAAAAAAAAAAA YOU JUST TOOK TEN YEARS OFF MY LIFE EXPECTANCY WITH YOUR ENTITLED BULLSHIT

do you actually think slightly misinformed documentation bug requests will kill you?

**Comment by CounterPillow** [comment]
>do you actually think slightly misinformed documentation bug requests will kill you?

Yes, and you just took another year off my life. If you keep this up you'll be facing a murder conviction.

**Comment by Hrxn** [comment]
I... don't get it. Could you maybe rephrase your issue a bit? Maybe I'll have a chance at understanding then.

You said that it was "impossible" to find a list of all options, but you actually managed to find them in the end? I don't get it. You have found the online manual page, you have found `--list-options`, what do you want? Already finished with reading, and now looking for even more documentation stuff to read?

**Comment by jimbo1qaz** [comment]
I couldn't find that on my own, and had to ask in a Discord chat room where someone told me the right option to use. Afterwards I spent quite some time looking for reasons I couldn't find it, and ways I could've found it on my own (apparently my research wasn't as thorough as I thought).

I should've just created a pull request in the first place, and expressed less frustration.

Anyway how do I link to sections in the readme? I tried a PR in https://github.com/mpv-player/mpv/pull/6562/files, but https://github.com/mpv-player/mpv/blob/e43b82072aa2416519e258ae5f86f125d98eac67/DOCS/man/mpv.rst#location-and-syntax doesn't look right.

Someone on Discord mentioned that you can add a sitemap to the README, to change the title displayed on Google, without changing the <title> tag. I'm not sure how, and think the <title> tag would be better if it said `mpv | readme/man`.
